src/react/inventory/sharedConnectorSetup.ts

```
@@ -0,0 +1,177 @@
+    return `${REMOTE_ASSETS}/item/${item.name ?? item.type}.png`
+  },
+  getBlockTextureUrl (item: ItemStack) {
+    return `${REMOTE_ASSETS}/block/${item.name ?? item.type}.png`
+  },
+}
+
+// ----- Window title formatter – resolves JSON text components to display strings -----
+
+/** Parse a raw window title (JSON string, NBT object, or plain text) into a readable string. */
+export function formatWindowTitle (rawTitle: any): string {
+  if (rawTitle == null) return ''
```

ESLint

[error] Use '===' to compare with null.

(no-eq-null)